src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -250,13 +251,30 @@ export const WorkspaceDashboard = _.flow(
           ]),
           ..._.map(({ membersGroupName }) => div({ style: styles.authDomain }, [membersGroupName]), authorizationDomain)
         ]),
-        div({ style: { margin: '1.5rem 0 0.5rem 0', borderBottom: `1px solid ${colors.gray[3]}` } }),
+        div({ style: { margin: '1.5rem 0 1rem 0', borderBottom: `1px solid ${colors.gray[3]}` } }),
+        div({ style: { fontSize: '1rem', fontWeight: 500, marginBottom: '0.5rem' } }, [
+          'Google Bucket',
+          h(Clickable, {
+            as: 'span',
+            style: { margin: '0 0.5rem', color: colors.green[0] },
+            tooltip: 'Copy bucket to clipboard',
+            onClick: async () => {
+              try {
+                await clipboard.writeText(bucketName)
+                this.setState({ bucketCopied: true }, () => {
+                  setTimeout(() => this.setState({ bucketCopied: undefined }), 1500)
```

Comment:
TOL: when we copy a notebook link to clipboard, a small toast appears. Do we want to mimic the behavior here instead of doing the icon timeout change?

Reply:
FWIW, I think I prefer the effect closer to the action like this is. A toast is more distracting than useful in this case.